scripts/plots/cam/exevs_refs_grid2obs_ctc_plots.sh

```diff
@@ -302,7 +302,7 @@ chmod +x run_all_poe.sh
 # Run the POE script in parallel or in sequence order to generate png files
 #**************************************************************************
 if [ $run_mpi = yes ] ; then
-   mpiexec -np 820 -ppn 82 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh
+   mpiexec -np 512 -ppn 86 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh
```

Comment:
The refs_grid2obs_ctc jobs are requesting `select=6:ncpus=86`, so 6 * 86 is 516 so need to adjust resources so the come together to 512